### DISCUSSION TITLE
allow to import non css module file inside component instead of global file like _app

### DISCUSSION THREAD

**Discussion by applemate** [vercel/next.js#34403] [Ideas]
### Describe the feature you'd like to request

allow to import non css module file inside component instead of global index

### Describe the solution you'd like

allow to import non css module file inside component instead of global index

### Describe alternatives you've considered

allow to import non css module file inside component instead of global index

**Comment by MonstraG** [comment]
https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules
> Since Next.js 9.5.4, importing a CSS file from node_modules is permitted anywhere in your application.

Is that what you're asking for?

**Comment by applemate** [comment]
In nextjs every css files and scss files are treated as global css. And the global css have to be put in the index.tsx or app.tsx. That what nextjs says and that's what nextjs does.

I want to put the css files (import) inside my components. Css module is not my option.

**Comment by andreisoare** [comment]
I'm interested in this as well. Here's a use case:

I'm using [Stream Chat](https://getstream.io/chat/docs/sdk/react/) in my NextJS app. They [make you include](https://getstream.io/chat/docs/sdk/react/customization/css_and_theming/) their global css, which is big ([191kB](https://unpkg.com/browse/stream-chat-css@1.0.17/dist/css/index.css)) and uses `@import"https://fonts.googleapis.com/css?...` which further blocks page rendering until these fonts download.

Because I can only include this global css in `_app.js`, it slows down all my pages, even the public ones that I need to be optimized for SEO.
